pkg/controllers/routing/network_routes_controller.go

```diff
@@ -756,7 +758,10 @@ func (nrc *NetworkRoutingController) startBgpServer() error {
 		asnStrings := stringToSlice(nodeBgpPeerAsnsAnnotation, ",")
 		peerASNs, err := stringSliceToUInt32(asnStrings)
 		if err != nil {
-			nrc.bgpServer.Stop()
+			err2 := nrc.bgpServer.Stop()
+			if err2 != nil {
+				return fmt.Errorf("Failed to stop bgpServer: %s", err)
```

Comment:
I think that in this case the parent error is more important than this error. I would log this and wait to return until below

Reply:
Same for all of the following handlings in this file too

Reply:
Done.